test/test_pyfftw_partial.py

```diff
@@ -53,9 +55,13 @@ def test_failure(self):
             if dtype not in _supported_types:
                 a = empty_aligned((1,1024), npdtype, n=16)
                 b = empty_aligned(a.shape, dtype=a.dtype, n=16)
-                with self.assertRaisesRegex(NotImplementedError, "Rebuild pyFFTW with support for %s precision!" % _all_types_human_readable[dtype]):
+                msg = "Rebuild pyFFTW with support for %s precision!" % _all_types_human_readable[dtype]
+                with self.assertRaisesRegex(NotImplementedError, msg):
                     FFTW(a,b)
 
+    def test_conversion(self):
+        # If double precision not supported, the builder should convert to single precision
```

Comment:
Are we assuming single precision is always supported? (also, you can put this as a docstring rather than a comment).

Reply:
I'm implementing the actual test, turned it into a docstring. And no, we don't assume single precision is always available, I changed the wording